src/Symfony/Component/Security/Http/RememberMe/PersistentTokenBasedRememberMeServices.php

```diff
@@ -29,6 +30,8 @@
  */
 class PersistentTokenBasedRememberMeServices extends AbstractRememberMeServices
 {
+    private const HASHED_TOKEN_PREFIX = 'hash_';
```

Comment:
Should we use `sha256_` as prefix, for the day we'll choose another algo?

Reply:
Sure. Updated.